src/Utils/AutoPilotUtils.ts

```diff
@@ -1,6 +1,6 @@
-export const minAutoPilotThroughput = 4000;
-
+export const minAutoPilotThroughput = 1000;
```

Comment:
please rename to autoPilotThroughput1K

Reply:
I think we should keep the current name for this one since it represents the minimum throughput value for autoscale. If we rename it to autoPilotThroughput1K then it loses the minimum part which is important for this number.